src/pages-and-resources/discussions/app-list/FeatureList.test.jsx

```diff
@@ -48,10 +48,12 @@ describe('FeaturesList', () => {
     const button = getByRole(container, 'button');
     userEvent.click(button);
     features.forEach((feature) => {
-      const featureNodes = queryAllByText(
-        container, messages[`featureName-${feature.id}`].defaultMessage,
-      );
-      expect(featureNodes.map(node => node.closest('div'))).toHaveLength(1);
+      if (app.featureIds.includes(feature.id)) {
+        const featureNodes = queryAllByText(
+          container, messages[`featureName-${feature.id}`].defaultMessage,
+        );
+        expect(featureNodes.map(node => node.closest('div'))).toHaveLength(1);
+      }
```

Comment:
as now you are only showing "supported features" - can you update this test to only loop on `app.featureIds` & check if those are displayed / rendered?